src/components/Card.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import './Card.css';
+
+class Card extends Component {
+
+  showImage = () => {
+    if (this.props.image_url) {
+      return (
+          <img className="card-image" src={this.props.image_url} alt="Inspiration" />
+      );
+    } else {
+    return (
+      <div />
+    );
+   }
+  }
+
+  render() {
+    return (
+      <div className="card">
+        <h2 className="centered title">{this.props.text}</h2>
+        <span
+          data-card-id={this.props.id}
+          onClick={(event) => this.props.deleteCardCallback(event.target.dataset.cardId)}
+          className="delete">×</span>
+        {this.showImage()}
```

Comment:
I really think that we shouldn't give them an option to put in Image URLs. If the URLs are broken, then the board will look really gross and bad and broken

The ideal feature for me would be: a dropdown with selects of five moods: love, applause, confetti, sun through clouds, and an explosion... and on select of those, when the card displays, it has that emoji displayed

I haven't estimated how much time that feature would take to build

Reply:
That's easy enough to add and we can have the emojis saved in the scaffold as an array.  I don't think it would take long to build